### ISSUE TITLE
No icons - latest Aeon Nox Silvo skin

### ISSUE THREAD

**Issue by Better-Red** [aitschti/plugin.video.cb20#2]
Hi, not sure whether this is something you can change or if it's an issue with the skin, but the thumbs don't show in the latest version of Aeon Nox Silvo (Kodi 21 Omega) from the developers GitHub. The thumbs work with SC19 and all other add-ons I've tried. Do you have any ideas? The CB20 thumbs do show in the version of Aeon Nox Silvo from the official repo. Many thanks.

**Comment by aitschti** [comment]
Hey, sorry for the late reply, but I cannot reproduce this, using Github skin (10.0.2) and Kodi 21 on Windows, all icons show (same way with the repo version). SC and CB do more or less show icons the same way, just from different URLs. I cannot help you sorry.

**Comment by Better-Red** [comment]
Thanks for getting back to me and for testing. Apologies, I should have been more specific. I've rolled back to Aeon Nox Silvo v10.0.2 but was using v10.0.4 when I saw the issue. I've tried again using Aeon Nox Silvo v10.0.4, SC19 v1.5.0 and CB20 v2.2.0 and while the thumbs and backgrounds show in SC19, they do not in CB20.

https://github.com/MikeSiLVO/skin.aeon.nox.silvo/archive/refs/heads/master.zip

**Comment by aitschti** [comment]
I tried v10.0.4 of the skin and works fine with CB20. CB20 never had any background images for the models, as CB doesn't show any on their profiles that I could use. Icons work fine, regardless of the version I tried. 

BUT, I may now see what your problem is. You are using list view, not icon view, am I right? If I am using list view instead of icon view, the thumbs do not show in the right top area of the skin? Is that it? This I can reproduce.

Seems the skin does not display the icon set with setArt('icon': ...) anymore. They now seem to use the 'poster' variable there. It works as expected then. I will push an update shortly setting the 'poster' variable as well in all locations. Seems like a good practise.

Thanks for the tip!
